src/core/symbology/qgsgraduatedsymbolrenderer.h

```diff
@@ -400,10 +435,7 @@ class CORE_EXPORT QgsGraduatedSymbolRenderer : public QgsFeatureRenderer
     //! \note not available in Python bindings
     static const char *graduatedMethodStr( GraduatedMethod method ) SIP_SKIP;
 
-    bool mUseSymmetricMode = false;
-    double mSymmetryPoint = 0.0;
-    QStringList mListForCboPrettyBreaks = QStringList();
-    bool mAstride = false;
+    std::shared_ptr<QgsClassificationMethod> mClassificationMethod;
```

Comment:
Why is this a shared pointer?

Reply:
As you re-run classify on the widget, a new method is created and assigned the renderer. I am worried that someone keeps on using it afterwards.

Reply:
I think in that case they should be taking a clone of it themselves.